src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidatorTest.java

```diff
@@ -162,11 +162,11 @@ void testErrorReportedWhenEmailFieldIsNull() {
 
     @Test
     void testErrorReportedWhenEmailFieldExceedsMaxLength() {
```

Comment:
can you also add test to check no error occurs when email is 256 chars

Reply:
Sure will add one now

Reply:
Added